[
    {
        "url": "https://stackoverflow.com/questions/18706041",
        "text": "macbook pro:~$ nova traceback most recent call last : file usr local bin nova , line in module from novaclient.shell import main file library python packages novaclient __init__.py , line in module import pbr.version importerror: no module named pbr.version",
        "where": 10
    },
    {
        "url": "https://stackoverflow.com/questions/18706041",
        "text": "macbook pro:~$ sudo python downloads pbr setup.py install traceback most recent call last : file setup.py , line in module file volumes spencerowen downloads pbr pbr util.py , line in cfg_to_args pbr.hooks.setup_hook config file volumes spencerowen downloads pbr pbr hooks __init__.py , line in setup_hook metadata_config.run file volumes spencerowen downloads pbr pbr hooks base.py , line in run self.hook file volumes spencerowen downloads pbr pbr hooks metadata.py , line in hook self.config name , self.config.get version , none file volumes spencerowen downloads pbr pbr packaging.py , line in get_version version _get_version_from_git pre_version file volumes spencerowen downloads pbr pbr packaging.py , line in _get_version_from_git git git git_dir describe always .replace file volumes spencerowen downloads pbr pbr packaging.py , line in _run_shell_command file system library frameworks python.framework versions subprocess.py line in __init__ errread, errwrite file system library frameworks python.framework versions subprocess.py line in _execute_child raise child_exception typeerror: must be encoded string without null bytes, not str",
        "where": 10
    },
    {
        "url": "https://stackoverflow.com/questions/19740109",
        "text": "req: curl i x post h content type: application json h accept: application json h user agent: python novaclient d auth : tenantname : admin , passwordcredentials : username : admin , password : admin_pass connectionpool:202 starting new http connection : controller debug connectionpool:296 post http resp: caseinsensitivedict date : sat, nov gmt , vary : x auth token , content length : , content type : application json resp body: access : token : issued_at : t , expires : t , id : , tenant : description : admin tenant , enabled : true, id : , name : admin , servicecatalog : endpoints : adminurl : region : regionone , internalurl : id : , publicurl : endpoints_links : , type : image , name : glance , endpoints : adminurl : region : regionone , internalurl : id : , publicurl : endpoints_links : , type : compute , name : nova , endpoints : adminurl : region : regionone , internalurl : id : , publicurl : endpoints_links : , type : identity , name : keystone , user : username : admin , roles_links : , id : , roles : name : admin , name : admin , metadata : is_admin : roles : req: curl i x get h x auth project id: admin h user agent: python novaclient h accept: application json h x auth token: connectionpool:202 starting new http connection : controller debug connectionpool:296 get http resp: caseinsensitivedict date : sat, nov gmt , content length : , content type : text plain , www authenticate : keystone resp body: authentication required req: curl i x post h content type: application json h accept: application json h user agent: python novaclient d auth : passwordcredentials : username : admin , password : admin_pass , tenantid : debug connectionpool:296 post http resp: caseinsensitivedict date : sat, nov gmt , vary : x auth token , content length : , content type : application json resp body: access : token : issued_at : t , expires : t , id : , tenant : description : admin tenant , enabled : true, id : , name : admin , servicecatalog : endpoints : adminurl : region : regionone , internalurl : id : , publicurl : endpoints_links : , type : image , name : glance , endpoints : adminurl : region : regionone , internalurl : id : , publicurl : endpoints_links : , type : compute , name : nova , endpoints : adminurl : region : regionone , internalurl : id : , publicurl : endpoints_links : , type : identity , name : keystone , user : username : admin , roles_links : , id : , roles : name : admin , name : admin , metadata : is_admin : roles : req: curl i x get h x auth project id: admin h user agent: python novaclient h accept: application json h x auth token: debug connectionpool:296 get http resp: caseinsensitivedict date : sat, nov gmt , content length : , content type : text plain , www authenticate : keystone resp body: authentication required debug shell:740 unauthorized http traceback most recent call last : file usr lib packages novaclient shell.py , line in main openstackcomputeshell .main map strutils.safe_decode, sys.argv file usr lib packages novaclient shell.py , line in main args.func self.cs, args file usr lib packages novaclient line in do_image_list image_list cs.images.list file usr lib packages novaclient line in list return self._list detail, query , images file usr lib packages novaclient base.py , line in _list _resp, body self.api.client.get url file usr lib packages novaclient client.py , line in get return self._cs_request url, get , file usr lib packages novaclient client.py , line in _cs_request raise unauthorized: unauthorized http error: unauthorized http",
        "where": 10
    },
    {
        "url": "https://stackoverflow.com/questions/20091298",
        "text": "httpclient httpclient new defaulthttpclient ; httppost httppost new httppost ; try jsonobject auth new jsonobject ; jsonobject json new jsonobject ; auth.put auth , json ; jsonobject pcredentials new jsonobject ; json.put passwordcredentials: , pcredentials ; pcredentials.put username , admin ; pcredentials.put password , admin ; log.i tag , passing your data ; stringentity new stringentity auth.tojsonstring ; utf application json ; httppost.setheader content type , application json ; httpresponse httppost .setentity ; execute http post request httpresponse response httpclient.execute httppost ; log.i tag , server response is ; catch ioexception e log.e tag , ioexception e.tostring ; todo auto generated catch block",
        "where": 9
    },
    {
        "url": "https://stackoverflow.com/questions/20982023",
        "text": "wget post data test wget: server returned error: http forbidden",
        "where": 10
    },
    {
        "url": "https://stackoverflow.com/questions/21105194",
        "text": "httpclient httpclient new defaulthttpclient ; httpput httpost new httpput mylink auth_fa6362c6520449f9a8905e84fee68f8c try httpost.setheader x auth token , ; httpost.setheader content type , application octet stream ; httpost.setheader content length , integer.tostring binaryimagelength ; httpost.setheader content disposition , binaryimage ; this is path of the image from the phone memory e.g. log.i , pushing your data ; httpresponse httpost ; responsehandler string responsehandler new basicresponsehandler ; string httpclient.execute httpost, responsehandler ; log.i , response.getstatusline ; log.i , ; catch ioexception e log.e , ioexception e.tostring ; todo auto generated catch block alertdialog alertdialog new alertdialog.builder mainactivity.this,alertdialog.theme_device_default_light .create ; alertdialog.settitle ; alertdialog.setmessage e.tostring ; alertdialog.setbutton ok , new dialoginterface.onclicklistener public void onclick dialoginterface dialog, int which todo add your code for the button here. ; alertdialog.show ;",
        "where": 9
    },
    {
        "url": "https://stackoverflow.com/questions/21171261",
        "text": "import static com.google.common.io.closeables.closequietly; import java.io.closeable; import java.util.set; import org.jclouds.contextbuilder; import org.jclouds.compute.computeservice; import org.jclouds.compute.computeservicecontext; import org.jclouds.compute.domain.computemetadata; import org.jclouds.logging.slf4j.config.slf4jloggingmodule; import org.jclouds.openstack.nova.v2_0.novaapi; import org.jclouds.openstack.nova.v2_0.novaasyncapi; import org.jclouds.openstack.nova.v2_0.domain.server; import org.jclouds.openstack.nova.v2_0.features.serverapi; import org.jclouds.rest.restcontext; import com.google.common.collect.immutableset; import com.google.inject.module; public class jcloudsopenstack implements closeable private computeservice compute; private restcontext nova; public static void main string args jcloudsopenstack jcloudopenstack new jcloudsopenstack ; try jcloudopenstack.init ; jcloudopenstack.listservers ; jcloudopenstack.close ; catch exception e e.printstacktrace ; finally jcloudopenstack.close ; private void init iterable modules immutableset. of new ; string provider openstack nova ; string identity ... ; login name string password ... ; password computeservicecontext context contextbuilder.newbuilder provider .credentials identity, password .modules modules .buildview computeservicecontext.class ; compute context.getcomputeservice ; nova context.unwrap ; private void listservers set extends computemetadata nodes compute.listnodes ; system.out.println nodes.size ; public void close closequietly compute.getcontext ;",
        "where": 9
    },
    {
        "url": "https://stackoverflow.com/questions/21647674",
        "text": "controller:~$ sudo keystone tenant create usage: keystone version timeout os username os password os tenant name os tenant id os auth url os region name os identity api version os token os endpoint os cacert insecure os cert os key os cache force new token stale duration ... keystone: error: unrecognized arguments: controller:35357",
        "where": 10
    },
    {
        "url": "https://stackoverflow.com/questions/22616974",
        "text": "nova flavor list error: errno connection refused",
        "where": 10
    },
    {
        "url": "https://stackoverflow.com/questions/22853557",
        "text": "public static void main string args throws ioexception blobstore blobstore .credentials test:test , test .buildview blobstorecontext.class .getblobstore ; blobstore.createcontainerinlocation null, container ; bytesource bytesource.wrap foo .getbytes charsets.utf_8 ; blob blobstore.blobbuilder foo bar ; system.out.println blobstore.putblob container , ; bytesource bytesource.wrap bar .getbytes charsets.utf_8 ; blob blobstore.blobbuilder foo bar ; system.out.println blobstore.putblob container , ; bytesource manifest bytesource.wrap .getbytes charsets.utf_8 ; todo: set manifest header here blob manifestblob blobstore.blobbuilder foo bar .payload manifest .contentlength manifest.size .build ; system.out.println blobstore.putblob container , manifestblob ; blob dloblob blobstore.getblob container , foo bar ; inputstream input dloblob.getpayload .openstream ; while true int input.read ; if i break; system.out.print char i ; should print foobar",
        "where": 9
    },
    {
        "url": "https://stackoverflow.com/questions/22948334",
        "text": "from ceilometer import service from ceilometer.api import app import sys from paste.exceptions.errormiddleware import errormiddleware sys.stdout sys.stderr initialize the oslo configuration library and logging service.prepare_service wsgi_app app.versionselectorapplication application errormiddleware show_exceptions_in_wsgi_errors=true,",
        "where": 9
    },
    {
        "url": "https://stackoverflow.com/questions/23699945",
        "text": "python test.py server: server: traceback most recent call last : file test.py , line in module print nova.volumes.list file usr lib packages novaclient line in list return self._list volumes query_string, volumes file usr lib packages novaclient base.py , line in _list _resp, body self.api.client.get url file usr lib packages novaclient client.py , line in get return self._cs_request url, get , file usr lib packages novaclient client.py , line in _cs_request file usr lib packages novaclient client.py , line in _time_request resp, body self.request url, method, file usr lib packages novaclient client.py , line in request raise exceptions.from_response resp, body, url, method novaclient.exceptions.notfound: not found http",
        "where": 10
    },
    {
        "url": "https://stackoverflow.com/questions/24814799",
        "text": "projects python np_orchestration.py traceback most recent call last : file np_orchestration.py , line in module obj.provisionnodes file np_orchestration.py , line in provisionnodes var obj.create self, file projects kenobi baremetal.py , line in create return self._create os baremetal nodes , body, node file opt stack python novaclient novaclient base.py , line in _create _resp, body self.api.client.post url, attributeerror: type object managerwithfind has no attribute client projects",
        "where": 10
    },
    {
        "url": "https://stackoverflow.com/questions/25526567",
        "text": "novaapi nova contextbuilder .newbuilder openstack nova .endpoint endpoint .credentials getidentity getpassword .modules modules .buildapi novaapi.class volumeattachmentapi attachment nova.getvolumeattachmentextensionforzone constant_region .get volumeattachment attachment.attachvolumetoserverasdevice volumeidexisting, instanceid, devicedevxvdb",
        "where": 3
    },
    {
        "url": "https://stackoverflow.com/questions/25600226",
        "text": "nova interface attach net id xxxx fixed ip test_a5 error: failed to attach interface http request id: xxx",
        "where": 10
    },
    {
        "url": "https://stackoverflow.com/questions/27880115",
        "text": "trace nova file usr lib packages sqlalchemy engine url.py , line in __init__ trace nova self.port int port trace nova valueerror: invalid literal for int with base",
        "where": 10
    },
    {
        "url": "https://stackoverflow.com/questions/27984258",
        "text": "import java.io.*; import java.net.*; import java.text.*; import java.util.*; public class stackorchestration private static final string token_id xananananana ; private static final string tenant_id ????? ; public static void createstack httpurlconnection connection null; bufferedreader br null; try url url new connection httpurlconnection url.openconnection ; string line; stringbuffer jsonstring new stringbuffer ; string payload stack_name\\ single server , disable_rollback\\ true, : , : heat_template_version: compute_instance: type: os::nova::server\\ properties:\\n flavor: gb image: centos pv name: single server public_ip:\\n description: public ip address of the deployed compute value: get_attr: compute_instance, , timeout_mins\\ ; connection.setdoinput true ; connection.setdooutput true ; connection.setrequestmethod post ; connection.setrequestproperty x auth token , token_id ; connection.setrequestproperty content type , application json; ; connection.setrequestproperty accept , application json ; outputstreamwriter writer new outputstreamwriter connection.getoutputstream , utf ; writer.write payload ; writer.close ; br new bufferedreader new inputstreamreader connection.getinputstream ; while line br.readline null system.out.println ; br.close ; connection.disconnect ; catch exception e inputstream httpurlconnection connection .geterrorstream ; br new bufferedreader new inputstreamreader ; string nachricht; string response ; try while nachricht br.readline null response nachricht; system.out.println response ; catch exception ex return; public static void main string args try createstack ; catch exception e e.printstacktrace ;",
        "where": 9
    },
    {
        "url": "https://stackoverflow.com/questions/28824639",
        "text": "try: opts dict tenant_id= tenant id value region_name= region name swift_conn swiftclient.client.connection auth url username password tenant_name= tenant name os_options=opts swift_conn.post_container cont swift_conn.close except swiftclient.exceptions.clientexception: print traceback.format_exc",
        "where": 9
    },
    {
        "url": "https://stackoverflow.com/questions/28968270",
        "text": "var sudo ip netns exec qrouter ping ping bytes of data. from icmp_seq=1 destination host unreachable",
        "where": 10
    },
    {
        "url": "https://stackoverflow.com/questions/28987303",
        "text": "for in hosts_list: try: hostname str h.host_name print try to get system from: hostname print cl.hosts.get hostname except exception as e: logger.error e",
        "where": 9
    },
    {
        "url": "https://stackoverflow.com/questions/29135484",
        "text": "curl i x post h content type: application json h accept: application json h user agent: python novaclient d auth : tenantname : xxxx , passwordcredentials : username : xxxx , password : xxxxx http connection established proxy agent: apache curl: ssl certificate problem, verify that the ca cert is ok. details: error:14090086:ssl routines:ssl3_get_server_certificate:certificate verify failed more details here: curl performs ssl certificate verification by default, using bundle of certificate authority ca public keys ca certs . if the bundle file isn t adequate, you can specify an alternate file using the cacert option. if this https server uses certificate signed by ca represented in the bundle, the certificate verification probably failed due to problem with the certificate it might be expired, or the name might not match the domain name in the url . if you d like to turn off curl s verification of the certificate, use the k or insecure option.",
        "where": 10
    },
    {
        "url": "https://stackoverflow.com/questions/29232988",
        "text": "import java.sql.*; public class userregistration static final string jdbc_driver com.mysql.jdbc.driver ; static final string db_url jdbc:mysql: localhost keystone ; static final string user root ; static final string pass ; public static void main string args connection conn null; statement stmt null; try class.forname com.mysql.jdbc.driver ; conn drivermanager.getconnection db_url,user,pass ; stmt conn.createstatement ; string sql; sql select name,id,password from user ; resultset rs stmt.executequery sql ; while rs.next string first rs.getstring name ; string last rs.getstring id ; string pwd rs.getstring password ; if siju .equals first if cusatse .equals pwd system.out.println login ok ; system.out.print username: first ; system.out.println , id: last ; system.out.println , password: pwd ; rs.close ; stmt.close ; conn.close ; catch sqlexception se se.printstacktrace ; catch exception e e.printstacktrace ; finally try if stmt!=null stmt.close ; catch sqlexception try if conn!=null conn.close ; catch sqlexception se se.printstacktrace ; system.out.println goodbye! ;",
        "where": 9
    },
    {
        "url": "https://stackoverflow.com/questions/31049369",
        "text": "trace nova from neutronclient.neutron import as trace nova importerror: no module named neutron",
        "where": 10
    },
    {
        "url": "https://stackoverflow.com/questions/31644241",
        "text": "trace nova programmingerror: programmingerror table nova.services does not exist",
        "where": 10
    },
    {
        "url": "https://stackoverflow.com/questions/33900693",
        "text": "inception opt cisco vms installer glance debug image delete curl g i x delete h accept encoding: gzip, deflate h accept: h user agent: python glanceclient h connection: keep alive h x auth token: h content type: application octet stream cert none key none request returned failure status traceback most recent call last : file usr local lib packages glanceclient shell.py , line in main args.func client, args file usr local lib packages glanceclient , line in do_image_delete gc.images.delete args.id file usr local lib packages glanceclient , line in delete self.http_client.delete url file usr local lib packages glanceclient common http.py , line in delete return self._request delete , url, file usr local lib packages glanceclient common http.py , line in _request resp, body_iter self._handle_response resp file usr local lib packages glanceclient common http.py , line in _handle_response raise exc.from_response resp, resp.content httpinternalservererror: httpinternalservererror http httpinternalservererror http",
        "where": 10
    },
    {
        "url": "https://stackoverflow.com/questions/33952706",
        "text": "usage: openstack service create h f shell,table c column variable variable prefix prefix name enable disable openstack service create: error: unrecognized arguments: description identity",
        "where": 10
    },
    {
        "url": "https://stackoverflow.com/questions/34075329",
        "text": "python python default, jun gcc red hat on type help , copyright , credits or license for more information. import import pyopenssl traceback most recent call last : file stdin , line in module importerror: no module named pyopenssl",
        "where": 10
    },
    {
        "url": "https://stackoverflow.com/questions/34137025",
        "text": "public static arraylist url getfilelist url arraylist url serverdir new arraylist ; try apacheurllister new apacheurllister ; serverdir.addall ; system.out.println serverdir ; catch exception e log.e getfilelist , listing url url ; e.printstacktrace ; return serverdir;",
        "where": 9
    },
    {
        "url": "https://stackoverflow.com/questions/34192462",
        "text": "openstack service create name keystone description openstack identity identity debug debug: cliff.commandmanager found command hypervisor_stats_show debug: cliff.commandmanager found command security_group_create debug: cliff.commandmanager found command security_group_rule_list debug: cliff.commandmanager found command ip_floating_add debug: cliff.commandmanager found command keypair_list debug: cliff.commandmanager found command flavor_unset debug: cliff.commandmanager found command host_show debug: cliff.commandmanager found command host_list debug: cliff.commandmanager found command hypervisor_show debug: cliff.commandmanager found command server_pause debug: cliff.commandmanager found command server_reboot debug: cliff.commandmanager found command server_migrate debug: cliff.commandmanager found command server_set debug: cliff.commandmanager found command server_add_security_group debug: cliff.commandmanager found command server_unrescue debug: cliff.commandmanager found command usage_list debug: cliff.commandmanager found command keypair_show debug: cliff.commandmanager found command security_group_set debug: cliff.commandmanager found command compute_agent_create debug: cliff.commandmanager found command server_rescue debug: cliff.commandmanager found command console_log_show debug: cliff.commandmanager found command compute_agent_delete debug: cliff.commandmanager found command server_ssh debug: cliff.commandmanager found command server_lock debug: cliff.commandmanager found command server_unset debug: cliff.commandmanager found command server_show debug: cliff.commandmanager found command server_suspend debug: cliff.commandmanager found command server_add_volume debug: cliff.commandmanager found command server_image_create debug: cliff.commandmanager found command flavor_list debug: cliff.commandmanager found command server_remove_volume debug: cliff.commandmanager found command compute_agent_set debug: cliff.commandmanager found command aggregate_add_host debug: cliff.commandmanager found command aggregate_remove_host debug: cliff.commandmanager found command compute_service_set debug: cliff.commandmanager found command aggregate_create debug: cliff.commandmanager found command keypair_create debug: cliff.commandmanager found command ip_floating_list debug: cliff.commandmanager found command aggregate_delete debug: cliff.commandmanager found command flavor_set debug: cliff.commandmanager found command security_group_rule_create debug: cliff.commandmanager found command security_group_delete debug: cliff.commandmanager found command server_rebuild debug: cliff.commandmanager found command flavor_delete debug: cliff.commandmanager found command server_delete debug: cliff.commandmanager found command project_usage_list debug: cliff.commandmanager found command availability_zone_list debug: cliff.commandmanager found command hypervisor_list debug: cliff.commandmanager found command flavor_create debug: cliff.commandmanager found command console_url_show debug: cliff.commandmanager found command ip_fixed_add debug: cliff.commandmanager found command server_remove_security_group debug: cliff.commandmanager found command usage_show debug: cliff.commandmanager found command compute_agent_list debug: cliff.commandmanager found command flavor_show debug: cliff.commandmanager found command ip_fixed_remove debug: cliff.commandmanager found command ip_floating_create debug: cliff.commandmanager found command server_list debug: cliff.commandmanager found command server_create debug: cliff.commandmanager found command ip_floating_pool_list debug: cliff.commandmanager found command aggregate_show debug: cliff.commandmanager found command security_group_show debug: cliff.commandmanager found command keypair_delete debug: cliff.commandmanager found command server_resize debug: cliff.commandmanager found command security_group_rule_delete debug: cliff.commandmanager found command ip_floating_delete debug: cliff.commandmanager found command ip_floating_remove debug: cliff.commandmanager found command security_group_list debug: cliff.commandmanager found command server_resume debug: cliff.commandmanager found command aggregate_set debug: cliff.commandmanager found command aggregate_list debug: cliff.commandmanager found command server_unpause debug: cliff.commandmanager found command compute_service_list debug: cliff.commandmanager found command server_unlock debug: openstackclient.shell compute api version cmd group openstack.compute.v2 debug: cliff.commandmanager found command network_set debug: cliff.commandmanager found command network_delete debug: cliff.commandmanager found command network_list debug: cliff.commandmanager found command network_show debug: cliff.commandmanager found command network_create debug: openstackclient.shell network api version cmd group openstack.network.v2 debug: cliff.commandmanager found command image_set debug: cliff.commandmanager found command image_delete debug: cliff.commandmanager found command image_create debug: cliff.commandmanager found command image_list debug: cliff.commandmanager found command image_show debug: cliff.commandmanager found command image_save debug: openstackclient.shell image api version cmd group openstack.image.v1 debug: cliff.commandmanager found command snapshot_show debug: cliff.commandmanager found command backup_create debug: cliff.commandmanager found command volume_list debug: cliff.commandmanager found command volume_show debug: cliff.commandmanager found command snapshot_unset debug: cliff.commandmanager found command volume_set debug: cliff.commandmanager found command backup_delete debug: cliff.commandmanager found command volume_create debug: cliff.commandmanager found command volume_type_list debug: cliff.commandmanager found command volume_type_create debug: cliff.commandmanager found command backup_restore debug: cliff.commandmanager found command backup_list debug: cliff.commandmanager found command volume_unset debug: cliff.commandmanager found command backup_show debug: cliff.commandmanager found command volume_type_delete debug: cliff.commandmanager found command volume_type_set debug: cliff.commandmanager found command snapshot_set debug: cliff.commandmanager found command snapshot_list debug: cliff.commandmanager found command snapshot_delete debug: cliff.commandmanager found command volume_delete debug: cliff.commandmanager found command snapshot_create debug: cliff.commandmanager found command volume_type_unset debug: openstackclient.shell volume api version cmd group openstack.volume.v1 debug: cliff.commandmanager found command project_create debug: cliff.commandmanager found command project_list debug: cliff.commandmanager found command debug: cliff.commandmanager found command service_list debug: cliff.commandmanager found command role_add debug: cliff.commandmanager found command project_show debug: cliff.commandmanager found command role_show debug: cliff.commandmanager found command endpoint_delete debug: cliff.commandmanager found command project_set debug: cliff.commandmanager found command service_create debug: cliff.commandmanager found command service_show debug: cliff.commandmanager found command endpoint_show debug: cliff.commandmanager found command debug: cliff.commandmanager found command catalog_list debug: cliff.commandmanager found command debug: cliff.commandmanager found command service_delete debug: cliff.commandmanager found command token_issue debug: cliff.commandmanager found command project_delete debug: cliff.commandmanager found command endpoint_list debug: cliff.commandmanager found command role_list debug: cliff.commandmanager found command user_create debug: cliff.commandmanager found command user_delete debug: cliff.commandmanager found command user_show debug: cliff.commandmanager found command role_create debug: cliff.commandmanager found command role_remove debug: cliff.commandmanager found command role_delete debug: cliff.commandmanager found command catalog_show debug: cliff.commandmanager found command token_revoke debug: cliff.commandmanager found command endpoint_create debug: cliff.commandmanager found command user_role_list debug: cliff.commandmanager found command user_set debug: cliff.commandmanager found command user_list debug: cliff.commandmanager found command debug: openstackclient.shell identity api version cmd group openstack.identity.v2 debug: cliff.commandmanager found command object_create debug: cliff.commandmanager found command object_list debug: cliff.commandmanager found command object_delete debug: cliff.commandmanager found command container_list debug: cliff.commandmanager found command object_show debug: cliff.commandmanager found command container_delete debug: cliff.commandmanager found command container_create debug: cliff.commandmanager found command container_show debug: cliff.commandmanager found command container_save debug: cliff.commandmanager found command object_save debug: openstackclient.shell object_store api version cmd group openstack.object_store.v1 debug: cliff.commandmanager found command extension_list debug: cliff.commandmanager found command quota_set debug: cliff.commandmanager found command quota_show debug: cliff.commandmanager found command limits_show info: openstackclient.shell command: none openstackclient.identity.v2_0.service.createservice debug: openstackclient.api.auth auth plugin token_endpoint selected debug: openstackclient.api.auth auth_type: token_endpoint debug: openstackclient.api.auth fetching option os_url debug: openstackclient.api.auth fetching option os_token info: openstackclient.common.clientmanager using auth plugin: token_endpoint debug: openstackclient.common.clientmanager get auth_ref debug: openstackclient.identity.v2_0.service.createservice take_action namespace openstack identity , table , max_width=0, keystone , , type_or_name= identity debug: openstackclient.identity.client instantiating identity client: class openstackclient.identity.client.identityclientv2 debug: keystoneclient.session req: curl g i x post h user agent: python keystoneclient h content type: application json h accept: application json h x auth token: d os ksadm:service : type : identity , name : keystone , description : openstack identity info: requests.packages.urllib3.connectionpool starting new http connection : debug: requests.packages.urllib3.connectionpool post http debug: keystoneclient.session resp: debug: keystoneclient.session request returned failure status: error: openstack the request you have made requires authentication. http request id: traceback most recent call last : file usr lib packages cliff app.py , line in run_subcommand result cmd.run parsed_args file usr lib packages cliff display.py , line in run column_names, data self.take_action parsed_args file usr lib packages openstackclient identity line in take_action parsed_args.description file usr lib packages keystoneclient line in create return self._create os ksadm services , body, os ksadm:service file usr lib packages keystoneclient base.py , line in _create return self._post url, body, response_key, return_raw, file usr lib packages keystoneclient base.py , line in _post resp, body self.client.post url, file usr lib packages keystoneclient adapter.py , line in post return self.request url, post , file usr lib packages keystoneclient adapter.py , line in request resp super legacyjsonadapter, self .request file usr lib packages keystoneclient adapter.py , line in request return self.session.request url, method, file usr lib packages keystoneclient utils.py , line in inner return func file usr lib packages keystoneclient session.py , line in request raise exceptions.from_response resp, method, url unauthorized: the request you have made requires authentication. http request id: debug: openstackclient.shell clean_up createservice debug: openstackclient.shell got an error: the request you have made requires authentication. http request id: error: openstackclient.shell traceback most recent call last : file usr lib packages openstackclient shell.py , line in run return super openstackshell, self .run argv file usr lib packages cliff app.py , line in run result self.run_subcommand remainder file usr lib packages cliff app.py , line in run_subcommand result cmd.run parsed_args file usr lib packages cliff display.py , line in run column_names, data self.take_action parsed_args file usr lib packages openstackclient identity line in take_action parsed_args.description file usr lib packages keystoneclient line in create return self._create os ksadm services , body, os ksadm:service file usr lib packages keystoneclient base.py , line in _create return self._post url, body, response_key, return_raw, file usr lib packages keystoneclient base.py , line in _post resp, body self.client.post url, file usr lib packages keystoneclient adapter.py , line in post return self.request url, post , file usr lib packages keystoneclient adapter.py , line in request resp super legacyjsonadapter, self .request file usr lib packages keystoneclient adapter.py , line in request return self.session.request url, method, file usr lib packages keystoneclient utils.py , line in inner return func file usr lib packages keystoneclient session.py , line in request raise exceptions.from_response resp, method, url unauthorized: the request you have made requires authentication. http request id:",
        "where": 10
    },
    {
        "url": "https://stackoverflow.com/questions/34389459",
        "text": "kolla ansible deploy deploying playbook ansible playbook i e etc kolla globals.yml e etc kolla passwords.yml error: merge_configs is not legal parameter in an ansible task or handler command failed ansible playbook i e etc kolla globals.yml e etc kolla passwords.yml",
        "where": 10
    },
    {
        "url": "https://stackoverflow.com/questions/35009233",
        "text": "openstack service create name keystone description openstack identity identity traceback most recent call last : file usr bin openstack , line in module from openstackclient.shell import main file usr lib packages openstackclient shell.py , line in module from openstackclient.common import clientmanager file usr lib packages openstackclient common clientmanager.py , line in module import requests file usr lib packages requests __init__.py , line in module from import utils file usr lib packages requests utils.py , line in module from .exceptions import invalidurl, filemodewarning importerror: cannot import name filemodewarning",
        "where": 10
    },
    {
        "url": "https://stackoverflow.com/questions/35146137",
        "text": "tools openstack traceback most recent call last : file usr bin openstack , line in module from openstackclient.shell import main file usr lib packages openstackclient shell.py , line in module from cliff import app file usr lib packages cliff app.py , line in module from .complete import completecommand file usr lib packages cliff complete.py , line in module from cliff import command file usr lib packages cliff command.py , line in module abc.abcmeta attributeerror: module object has no attribute add_metaclass",
        "where": 10
    },
    {
        "url": "https://stackoverflow.com/questions/35347057",
        "text": "osclient os osfactory.builderv3 .endpoint auth_url .credentials username, password .scopetoproject projectident, domainident .authenticate ; exception thrown by application class org.openstack4j.connectors.jersey2.httpexecutorserviceimpl.invoke:58 org.openstack4j.api.exceptions.connectionexception: org.apache.cxf.interceptor.fault: no message body writer has been found for class org.openstack4j.openstack.identity.domain.v3.keystoneauth, contenttype: application json at org.openstack4j.connectors.jersey2.httpexecutorserviceimpl.invoke at org.openstack4j.connectors.jersey2.httpexecutorserviceimpl.execute at org.openstack4j.core.transport.internal.httpexecutor.execute at org.openstack4j.openstack.internal.osauthenticator.authenticatev3 at org.openstack4j.openstack.internal.osauthenticator.invoke at org.openstack4j.openstack.client.osclientbuilder$clientv3.authenticate at org.openstack4j.openstack.client.osclientbuilder$clientv3.authenticate at com.ibm.sample.objectstorageservice. objectstorageservice.java:36 at wasdev.sample.servlet.simpleservlet.doget simpleservlet.java:30 at javax.servlet.http.httpservlet.service httpservlet.java:687 at javax.servlet.http.httpservlet.service httpservlet.java:790 at com.ibm.ws.webcontainer.servlet.servletwrapper.service servletwrapper.java:1290 at internal classes caused by: javax.ws.rs.processingexception: org.apache.cxf.interceptor.fault: no message body writer has been found for class org.openstack4j.openstack.identity.domain.v3.keystoneauth, contenttype: application json at org.apache.cxf.jaxrs.client.webclient.doresponse webclient.java:1206 at internal classes at org.openstack4j.connectors.jersey2.httpcommand.execute at org.openstack4j.connectors.jersey2.httpexecutorserviceimpl.invokerequest at org.openstack4j.connectors.jersey2.httpexecutorserviceimpl.invoke ... more caused by: org.apache.cxf.interceptor.fault: no message body writer has been found for class org.openstack4j.openstack.identity.domain.v3.keystoneauth, contenttype: application json at org.apache.cxf.jaxrs.client.webclient$bodywriter.dowritebody webclient.java:1286 ... more caused by: javax.ws.rs.processingexception: no message body writer has been found for class org.openstack4j.openstack.identity.domain.v3.keystoneauth, contenttype: application json at org.apache.cxf.jaxrs.client.abstractclient.reportmessagehandlerproblem abstractclient.java:758 ... more caused by: org.codehaus.jackson.map.jsonmappingexception: was java.lang.nullpointerexception through reference chain: org.openstack4j.openstack.identity.domain.v3.keystoneauth at org.codehaus.jackson.map.jsonmappingexception.wrapwithpath jsonmappingexception.java:215 ... more caused by: java.lang.nullpointerexception: at org.openstack4j.openstack.identity.domain.v3.keystoneauth.getid at sun.reflect.nativemethodaccessorimpl.invoke0 native method at sun.reflect.nativemethodaccessorimpl.invoke unknown source at sun.reflect.delegatingmethodaccessorimpl.invoke unknown source at java.lang.reflect.method.invoke unknown source at org.codehaus.jackson.map.ser.beanpropertywriter.get beanpropertywriter.java:285",
        "where": 9
    },
    {
        "url": "https://stackoverflow.com/questions/35439056",
        "text": "import java.io.closeable; import java.io.ioexception; import java.util.set; import org.jclouds.contextbuilder; import org.jclouds.openstack.swift.v1.swiftapi; import org.jclouds.openstack.swift.v1.domain.container; import org.jclouds.openstack.swift.v1.features.containerapi; import com.google.common.io.closeables; public class jcloudsswift implements closeable private swiftapi swiftapi; public static void main string args jcloudsswift jcloudsswift new jcloudsswift ; try jcloudsswift.listcontainers ; jcloudsswift.close ; catch exception e e.printstacktrace ; finally try jcloudsswift.close ; catch ioexception e e.printstacktrace ; public jcloudsswift string provider openstack swift ; string identity tenantname:username ; string credential password ; string endpoint swiftapi contextbuilder.newbuilder provider .endpoint endpoint .credentials identity, credential .buildapi swiftapi.class ; private void listcontainers string region us east ; containerapi containerapi swiftapi.getcontainerapi region ; set container containers containerapi.list .toset ; for container container containers system.out.println container ; public void close throws ioexception closeables.close swiftapi, true ;",
        "where": 9
    },
    {
        "url": "https://stackoverflow.com/questions/35836743",
        "text": "sudo ip netns exec qrouter ping ping bytes of data. from icmp_seq=1 destination host unreachable from icmp_seq=2 destination host unreachable from icmp_seq=3 destination host unreachable from icmp_seq=4 destination host unreachable ping statistics packets transmitted, received, errors, packet loss, time pipe",
        "where": 10
    },
    {
        "url": "https://stackoverflow.com/questions/35910437",
        "text": "keystonerc_admin:admin cinder insecure quota defaults insecurerequestwarning: unverified https request is being made. adding certificate verification is strongly advised. see: insecurerequestwarning error: the server has either erred or is incapable of performing the requested operation. http request id:",
        "where": 10
    },
    {
        "url": "https://stackoverflow.com/questions/38729696",
        "text": "private void copy string origincontainer, string origin, string destinationcontainer, string destination try url url new url storageurl delimiter destinationcontainer delimiter destination ; httpurlconnection conn httpurlconnection url.openconnection ; conn.setdoinput true ; conn.setdooutput true ; conn.setrequestmethod put ; conn.setrequestproperty x auth token , authtoken ; conn.setrequestproperty x copy from , delimiter origincontainer delimiter origin ; conn.setrequestproperty content length , ; if conn.getresponsecode httpurlconnection.http_created system.err.println while copying the object: conn.getresponsecode ; conn.disconnect ; catch malformedurlexception e system.err.println while copying the object: e.getmessage ; catch ioexception e system.err.println while copying the object: e.getmessage ;",
        "where": 9
    },
    {
        "url": "https://stackoverflow.com/questions/42934390",
        "text": "sudo h http_proxy= https_proxy= no_proxy= pip_find_links= setuptools_sys_path_technique=rewrite install c e ignoring markers python_version don t match your environment obtaining file: opt stack glance exception: traceback most recent call last : file usr local lib packages pip basecommand.py , line in main status self.run options, args file usr local lib packages pip commands install.py , line in run wb.build file usr local lib packages pip wheel.py , line in build self.requirement_set.prepare_files self.finder file usr local lib packages pip req_set.py , line in prepare_files ignore_dependencies=self.ignore_dependencies file usr local lib packages pip req_set.py , line in _prepare_file req_to_install.check_if_exists file usr local lib packages pip req_install.py , line in check_if_exists self.req.name file usr local lib packages pip _vendor pkg_resources __init__.py , line in get_distribution dist get_provider dist file usr local lib packages pip _vendor pkg_resources __init__.py , line in get_provider return working_set.find moduleorreq or require str moduleorreq file usr local lib packages pip _vendor pkg_resources __init__.py , line in require needed self.resolve parse_requirements requirements file usr local lib packages pip _vendor pkg_resources __init__.py , line in resolve raise versionconflict dist, .with_context dependent_req contextualversionconflict: usr local lib packages , requirement.parse set castellan python:pip_install:1 exit_trap stack.sh:exit_trap:487 local stack.sh:exit_trap:488 jobs p stack.sh:exit_trap:488 stack.sh:exit_trap:491 n stack.sh:exit_trap:497 kill_spinner stack.sh:kill_spinner:383 ! z stack.sh:exit_trap:499 ne stack.sh:exit_trap:500 echo on exit on exit stack.sh:exit_trap:501 generate subunit fail stack.sh:exit_trap:502 z stack.sh:exit_trap:505 d stack.sh:exit_trap:511 exit",
        "where": 10
    },
    {
        "url": "https://stackoverflow.com/questions/43971617",
        "text": "test.py centos traceback most recent call last : file test.py , line in module print nova.glance.find_image sys.argv file usr local lib packages novaclient , line in find_image images file usr local lib packages novaclient base.py , line in _list resp, body self.api.client.get url file usr local lib packages , line in get return self.request url, get , file usr local lib packages novaclient client.py , line in request raise exceptions.from_response resp, body, url, method novaclient.exceptions.badrequest: unable to filter by unknown operator centos br br http",
        "where": 10
    },
    {
        "url": "https://stackoverflow.com/questions/43971617",
        "text": "test.py centos traceback most recent call last : file test.py , line in module print nova.glance.find_image sys.argv file usr local lib packages novaclient , line in find_image raise exceptions.notfound msg novaclient.exceptions.notfound: no image matching centos http",
        "where": 10
    },
    {
        "url": "https://stackoverflow.com/questions/44307815",
        "text": "ldap_url_parse_ext ldap: localhost ldap_init: trying ldap_init: using ldap_init: home env is ldap_init: trying ldap_init: trying ldap_init: trying ldaprc ldap_init: ldapconf env is null ldap_create ldap_url_parse_ext ldap: ldap_sasl_bind ldap_send_initial_request ldap_new_connection ldap_int_open_connection ldap_connect_to_host: tcp ldap_new_socket: ldap_prepare_socket: ldap_connect_to_host: trying ldap_pvt_connect: fd: tm: async: attempting to connect: connect success ldap_open_defconn: successful ldap_send_server_request ldap_result ld msgid ld msgid infinite timeout continue ld msgid all ld connections: host: port: refcnt: status: connected last used: thu jun ld outstanding requests: msgid origid status inprogress outstanding referrals parent count ld request count abandoned ld response queue: empty ld response count ldap_chkresponselist ld msgid all ldap_chkresponselist returns ld null ldap_int_select ld msgid all ld msgid message type bind ld new referrals mark request completed, ld msgid request done: ld msgid res_errno: res_error: res_matched: ldap_free_request origid msgid ldap_parse_result ldap_msgfree ldap_search_ext put_filter: put_filter: and put_filter_list put_filter: put_filter: simple put_simple_filter: put_filter: put_filter: simple put_simple_filter: put_filter: put_filter: simple put_simple_filter: ldap_build_search_req attrs: cn userpassword enabled sn mail description ldap_send_initial_request ldap_send_server_request ldap_result ld msgid ld msgid infinite timeout continue ld msgid all ld connections: host: port: refcnt: status: connected last used: thu jun ld outstanding requests: msgid origid status inprogress outstanding referrals parent count ld request count abandoned ld response queue: empty ld response count ldap_chkresponselist ld msgid all ldap_chkresponselist returns ld null ldap_int_select ld msgid all ld msgid message type search result ld new referrals mark request completed, ld msgid request done: ld msgid res_errno: res_error: res_matched: ldap_free_request origid msgid ldap_parse_result ldap_msgfree keystone.auth.plugins.core could not find user: admin keystone.common.wsgi authorization failed. could not find user: admin disable insecure_debug mode to suppress these",
        "where": 9
    },
    {
        "url": "https://stackoverflow.com/questions/44986818",
        "text": "stack_id str hc.stacks.get stack_name .id hc.stacks.delete stack_id=stack_id try: evntsdata hc.events.list stack_name .to_dict event_handle evntsdata resource_status if event_handle delete_in_progress : loopcontinue true while loopcontinue: evntsdata hc.events.list stack_name .to_dict event_handle evntsdata resource_status if event_handle delete_complete : loopcontinue false print str timestamp delete is completed! elif event_handle delete_failed : print failed this needs proper msg sys.exit else: print str timestamp delete in progress! time.sleep elif event_handle delete_complete : print str timestamp delete is completed! sys.exit elif event_handle delete_failed : print failed sys.exit except attributeerror as e: print str timestamp error: stack delete failure raise except runtimeerror, heatclient.exc.notfound : print stack doesnt exist: , stack_name",
        "where": 9
    },
    {
        "url": "https://stackoverflow.com/questions/45013893",
        "text": "public void createinstance string instancename for string region regions serverapi serverapi novaapi.getserverapi region ; system.out.println server api serverapi.tostring ; string imageid ; string flavorid ; an servercreated ser serverapi.create instancename, imageid, flavorid ; server server serverapi.get ser.getid ; while server.getstatus .value .equals active false try thread.sleep catch interruptedexception ex server serverapi.get ser.getid ; system.out.println ser.getid ;",
        "where": 9
    },
    {
        "url": "https://stackoverflow.com/questions/45915511",
        "text": "keystone_admin nova reboot poll cannot reboot instance while it is in vm_state http request id: commanderror : unable to reboot the specified server s . keystone_admin nova reboot hard poll request to reboot server server: dsc has been accepted. rebooting server commanderror : wait for specified server s failed.",
        "where": 10
    },
    {
        "url": "https://stackoverflow.com/questions/46068395",
        "text": "from openstack import connection import openstack conn project_name= admin test conn.identity.v2._proxy.create_role",
        "where": 3
    },
    {
        "url": "https://stackoverflow.com/questions/48746200",
        "text": "command: sudo designate rootwrap rndc s p c k addzone corp.syedinttest11.org type slave; masters port ; file slave.corp.syedinttest11.org.faaf39d3 ; exit code: stdout: u stderr: u rndc: addzone failed: permission trace designate.pool_manager.service traceback most recent call last : trace designate.pool_manager.service file opt stack designate designate pool_manager service.py , line in _create_zone_on_target trace designate.pool_manager.service backend.create_zone context, zone trace designate.pool_manager.service file opt stack designate designate backend impl_bind9.py line in create_zone trace designate.pool_manager.service self._execute_rndc rndc_op trace designate.pool_manager.service file opt stack designate designate backend impl_bind9.py line in _execute_rndc trace designate.pool_manager.service raise exceptions.backend e trace designate.pool_manager.service backend: unexpected while running command. trace designate.pool_manager.service command: sudo designate rootwrap rndc s p c k addzone corp.syedinttest11.org type slave; masters port ; file slave.corp.syedinttest11.org.faaf39d3 ; trace designate.pool_manager.service exit code: trace designate.pool_manager.service stdout: u trace designate.pool_manager.service stderr: u rndc: addzone failed: permission",
        "where": 10
    },
    {
        "url": "https://stackoverflow.com/questions/48786534",
        "text": "heat controller sudo ip netns exec qdhcp ping ping bytes of data. from icmp_seq=1 destination host unreachable from icmp_seq=2 destination host unreachable from icmp_seq=3 destination host unreachable from icmp_seq=4 destination host unreachable ping statistics packets transmitted, received, errors, packet loss, time pipe",
        "where": 10
    },
    {
        "url": "https://stackoverflow.com/questions/48786534",
        "text": "heat controller sudo ip netns exec qrouter ping ping bytes of data. from icmp_seq=1 destination host unreachable from icmp_seq=2 destination host unreachable from icmp_seq=3 destination host unreachable from icmp_seq=4 destination host unreachable stopped sudo ip netns exec qrouter ping",
        "where": 10
    },
    {
        "url": "https://stackoverflow.com/questions/49524302",
        "text": "nova boot config drive true flavor image file user data cloudinitfile test usage: nova version debug os cache timings os region name region name service type service type service name service name os endpoint type endpoint type os compute api version compute api ver endpoint override bypass url profile hmac_key insecure os cacert ca certificate os cert certificate os key key timeout seconds os auth type name os auth url os_auth_url os system scope os_system_scope os domain id os_domain_id os domain name os_domain_name os project id os_project_id os project name os_project_name os project domain id os_project_domain_id os project domain name os_project_domain_name os trust id os_trust_id os domain id os_default_domain_id os domain name os_default_domain_name os user id os_user_id os username os_username os user domain id os_user_domain_id os user domain name os_user_domain_name os password os_password subcommand ... error: unrecognized arguments: file test try nova help for more information.",
        "where": 10
    },
    {
        "url": "https://stackoverflow.com/questions/50873500",
        "text": "curl header x auth token: blah x post d name : swarm , master_count discovery_url :null, cluster_template_id : node_count keypair gitlab ci , master_flavor_id :null, labels : , flavor_id :null errors : status : code : client , links : , title : invalid input for field attribute cluster , detail : invalid input for field attribute cluster. value: master_count\\ discovery_url\\ cluster_template_id\\ node_count\\ gitlab master_flavor_id\\ :null,\\n flavor_id\\ unable to convert to cluster. error: __init__ takes exactly argument given , request_id :",
        "where": 10
    },
    {
        "url": "https://stackoverflow.com/questions/51076444",
        "text": "from openstack import connection conn",
        "where": 3
    },
    {
        "url": "https://stackoverflow.com/questions/51086374",
        "text": "openstack exception raised: when using gi.repository you must not import static modules like gobject . please change all occurrences of import gobject to from gi.repository import gobject . see:",
        "where": 10
    },
    {
        "url": "https://stackoverflow.com/questions/51371862",
        "text": "log opt python v python log opt log opt pip v pip from python log opt log opt log opt python constraint constraint pip setuptools wheel ignoring jsbeautifier: markers python_version== don t match your environment ignoring jsbeautifier: markers python_version== don t match your environment ignoring hyperframe: markers python_version== don t match your environment ignoring hyperframe: markers python_version== don t match your environment ignoring cssutils: markers python_version== don t match your environment ignoring cssutils: markers python_version== don t match your environment ignoring kaitaistruct: markers python_version== don t match your environment ignoring kaitaistruct: markers python_version== don t match your environment ignoring tornado: markers python_version== don t match your environment ignoring tornado: markers python_version== don t match your environment exception: traceback most recent call last : file tmp pip.zip pip basecommand.py , line in main status self.run options, args file tmp pip.zip pip commands install.py , line in run wheel_cache file tmp pip.zip pip basecommand.py , line in populate_requirement_set requirement_set.add_requirement file tmp pip.zip pip req_set.py , line in add_requirement return result unboundlocalerror: local variable result referenced before assignment log opt",
        "where": 10
    },
    {
        "url": "https://stackoverflow.com/questions/53141473",
        "text": "import openstack conn openstack.connect conn.list_flavors",
        "where": 3
    },
    {
        "url": "https://stackoverflow.com/questions/53472463",
        "text": "python userwarning: using keystoneclient sessions has been deprecated. please update your software to use warnings.warn using keystoneclient sessions has been deprecated. traceback most recent call last : file tmp o.py , line in module nova.flavors.list file usr lib packages novaclient , line in list return self._list detail, flavors , file usr lib packages novaclient base.py , line in _list resp, body self.api.client.get url file usr lib packages , line in get return self.request url, get , file usr lib packages novaclient client.py , line in request file usr lib packages , line in request resp super legacyjsonadapter, self .request file usr lib packages , line in request return self.session.request url, method, file usr lib packages keystoneclient session.py , line in request resp send file usr lib packages keystoneclient session.py , line in _send_request raise exceptions.sslerror msg ssl exception connecting to httpsconnectionpool : max retries exceeded with url: caused by sslerror sslerror bad handshake: syscallerror econnreset , ,",
        "where": 10
    },
    {
        "url": "https://stackoverflow.com/questions/53566189",
        "text": "cinder create display name filestorage error: version is not supported by the api. minimum is and maximum is http request id:",
        "where": 10
    },
    {
        "url": "https://stackoverflow.com/questions/54142698",
        "text": "openstack image create file gammy_image.iso jacobs gammy image public forbidden you are not authorized to complete publicize_image action. http",
        "where": 10
    },
    {
        "url": "https://stackoverflow.com/questions/54407965",
        "text": "show error: unable to establish connection to connection aborted. , badstatusline ,",
        "where": 10
    },
    {
        "url": "https://stackoverflow.com/questions/57306785",
        "text": "openstack orchestration service list warning: failed to import plugin clustering. error: none",
        "where": 10
    },
    {
        "url": "https://stackoverflow.com/questions/57985093",
        "text": "kolla ansible i . all in one deploy fatal: localhost : failed! changed : true, cmd : docker , update , restart , no , mariadb , delta : end : , msg : non zero return code , rc : start : , stderr : response from daemon: no such container: mariadb , stderr_lines : response from daemon: no such container: mariadb , stdout : , stdout_lines :",
        "where": 10
    },
    {
        "url": "https://stackoverflow.com/questions/58219679",
        "text": "openstack project create domain description service project service failed to discover available identity versions when contacting attempting to parse version from url. unable to establish connection to httpconnectionpool controller , : max retries exceeded with url: caused by newconnectionerror object at failed to establish new connection: errno nom ou service inconnu , tail keystone.access_rules_config.backends.json no config file found for access rules, application credential access rules will be unavailable.: ioerror: errno no such file or directory: etc keystone access_rules.json",
        "where": 10
    },
    {
        "url": "https://stackoverflow.com/questions/61666481",
        "text": "sudo microstack.init auto microstack_init configuring networking ... microstack_init setting up forwarding... microstack_init opening horizon dashboard up to microstack_init waiting for rabbitmq to start ... waiting for microstack_init rabbitmq started! microstack_init configuring rabbitmq ... microstack_init rabbitmq configured! microstack_init waiting for mysql server to start ... waiting for microstack_init mysql server started! creating databases ... warning: can t create database neutron ; database exists result self._query query warning: using grant statement to modify existing user s properties other than privileges is deprecated and will be removed in future release. use alter user statement for this operation. result self._query query warning: can t create database nova ; database exists result self._query query warning: can t create database nova_api ; database exists result self._query query warning: can t create database nova_cell0 database exists result self._query query warning: can t create database cinder ; database exists result self._query query warning: can t create database glance ; database exists result self._query query warning: can t create database keystone ; database exists result self._query query traceback most recent call last : file snap microstack bin microstack_init , line in module load_entry_point microstack , console_scripts , microstack_init file snap microstack lib packages init main.py , line in main question.ask file snap microstack lib packages init questions question.py , line in ask self.yes awr file snap microstack lib packages init questions __init__.py , line in yes check snapctl , start , microstack.nginx file snap microstack lib packages init shell.py , line in check raise subprocess.calledprocesserror proc.returncode, .join args subprocess.calledprocesserror: command snapctl start microstack.nginx returned non zero exit status",
        "where": 10
    },
    {
        "url": "https://stackoverflow.com/questions/61909348",
        "text": "batchdelete ; fatal error: uncaught runtimeexception: is not defined",
        "where": 10
    },
    {
        "url": "https://stackoverflow.com/questions/61909348",
        "text": "bulkdelete ; fatal error: uncaught runtimeexception: is not defined",
        "where": 10
    },
    {
        "url": "https://stackoverflow.com/questions/63613821",
        "text": "var sudo microstack.init auto sudo passwort microstack_init configuring networking ... microstack_init opening horizon dashboard up to microstack_init waiting for rabbitmq to start ... waiting for microstack_init rabbitmq started! microstack_init configuring rabbitmq ... microstack_init rabbitmq configured! microstack_init waiting for mysql server to start ... waiting for microstack_init mysql server started! creating databases ... warning: can t create database neutron ; database exists result self._query query warning: using grant statement to modify existing user s properties other than privileges is deprecated and will be removed in future release. use alter user statement for this operation. result self._query query warning: can t create database nova ; database exists result self._query query warning: can t create database nova_api ; database exists result self._query query warning: can t create database nova_cell0 database exists result self._query query warning: can t create database cinder ; database exists result self._query query warning: can t create database glance ; database exists result self._query query warning: can t create database keystone ; database exists result self._query query traceback most recent call last : file snap microstack bin microstack_init , line in module sys.exit load_entry_point microstack , console_scripts , microstack_init file snap microstack lib packages init main.py , line in wrapper return func file snap microstack lib packages init main.py , line in init question.ask file snap microstack lib packages init questions question.py , line in ask self.yes awr file snap microstack lib packages init questions __init__.py , line in yes check snapctl , start , microstack.nginx file snap microstack lib packages init shell.py , line in check raise subprocess.calledprocesserror proc.returncode, .join args subprocess.calledprocesserror: command snapctl start microstack.nginx returned non zero exit status var netstat grep tcp metropolis:55384 verbunden tcp metropolis:48726 verbunden tcp metropolis:59164 verbunden tcp metropolis:50820 verbunden var netstat grep tcp metropolis:44324 time_wait tcp metropolis:44376 time_wait var",
        "where": 10
    },
    {
        "url": "https://stackoverflow.com/questions/63749573",
        "text": "import openstack conn openstack.connection.connection servers conn.compute.servers all_projects=true",
        "where": 3
    },
    {
        "url": "https://stackoverflow.com/questions/64390511",
        "text": "cinder reset state state available warning:cinderclient.shell:api version requested, warning:cinderclient.shell:downgrading to based on server support. policy doesn t allow volume_extension:volume_admin_actions:reset_status to be performed. http request id: error: unable to reset the state for the specified entity s .",
        "where": 10
    },
    {
        "url": "https://stackoverflow.com/questions/64599843",
        "text": "protected long segmentationsize protected long currentsegment private inputstream inputstream; supplied externally private long inputstreamsize; supplied externally public void uploadsegmentedobjects inputstream segmentstream getnextsegment ; while segmentstream null do something public inputstream getnextsegment if done return null; inputstream segment createsegment ; return segment; protected boolean done return currentsegment segmentationsize inputstreamsize; protected inputstream createsegment throws ioexception boundedinputstream stream new boundedinputstream inputstream, segmentationsize ; stream.setpropagateclose false ; return stream;",
        "where": 9
    },
    {
        "url": "https://stackoverflow.com/questions/66953569",
        "text": "systemctl status neutron dhcp agent neutron dhcp agent.service openstack neutron dhcp agent loaded: loaded usr lib systemd system neutron dhcp agent.service; enabled; vendor preset: disabled active: active running since mon pdt; ago main pid: usr bin python tasks: limit: memory: cgroup: config file config file apr kbshostname neutron dhcp agent : privsep apr kbshostname sudo : neutron usr bin neutron rootwrap apr kbshostname neutron dhcp agent : critical privsep unhandled error: permissionerror: apr kbshostname neutron dhcp agent : privsep traceback most recent call last : apr kbshostname neutron dhcp agent : privsep file bin privsep helper , line in apr kbshostname neutron dhcp agent : privsep sys.exit helper_main apr kbshostname neutron dhcp agent : privsep file usr lib packages o apr kbshostname neutron dhcp agent : privsep sock.connect cfg.conf.privsep_sock_pat apr kbshostname neutron dhcp agent : privsep permissionerror: errno permission den apr kbshostname neutron dhcp agent : privsep lines end",
        "where": 10
    },
    {
        "url": "https://stackoverflow.com/questions/67418448",
        "text": "using python to install setuptools!=24.0.0,!=34.0.0,!=34.0.1,!=34.0.2,!=34.0.3,!=34.1.0,!=34.1.1,!=34.2.0,!=34.3.0,!=34.3.1,!=34.3.2,!=36.2.0,!=48.0.0,!=49.0.0 inc python:pip_install:187 sudo h lc_all=en_us.utf http_proxy= https_proxy= no_proxy= pip_find_links= setuptools_sys_path_technique=rewrite m pip install c ignore installed setuptools!=24.0.0,!=34.0.0,!=34.0.1,!=34.0.2,!=34.0.3,!=34.1.0,!=34.1.1,!=34.2.0,!=34.3.0,!=34.3.1,!=34.3.2,!=36.2.0,!=48.0.0,!=49.0.0 deprecation: constraints are only allowed to take the form of package name and version specifier. other forms were originally permitted as an accident of the implementation, but were undocumented. the new implementation of the resolver no longer supports these forms. possible replacement is replacing the constraint with requirement.. you can find discussion regarding this at error: editable requirements are not allowed as constraints stack.sh:main:752 err_trap stack.sh:err_trap:530 local stack.sh failed: full log in on exit",
        "where": 10
    },
    {
        "url": "https://stackoverflow.com/questions/68180947",
        "text": "cinder list id status name size volume type bootable attached to errors in cinder.volume.targets.tgt failed recovery attempt to create iscsi backing lun for volume id:iqn.2010 unexpected while running command. command: sudo cinder rootwrap tgtadm lld iscsi op new mode logicalunit tid lun b exit code: stdout: u stderr: u tgtadm: invalid cinder.volume.manager failed to re export volume, setting to error. cinder.volume.manager traceback most recent call last : cinder.volume.manager file usr lib packages cinder volume manager.py , line in init_host cinder.volume.manager self.driver.ensure_export ctxt, volume cinder.volume.manager file usr lib packages cinder volume drivers lvm.py , line in ensure_export cinder.volume.manager self.target_driver.ensure_export context, volume, volume_path cinder.volume.manager file usr lib packages cinder volume targets iscsi.py , line in ensure_export cinder.volume.manager old_name=none, cinder.volume.manager file usr lib packages cinder utils.py , line in _wrapper cinder.volume.manager return r.call f, cinder.volume.manager file usr lib packages retrying.py , line in call cinder.volume.manager return attempt.get self._wrap_exception cinder.volume.manager file usr lib packages retrying.py , line in get cinder.volume.manager six.reraise self.value , self.value self.value cinder.volume.manager file usr lib packages retrying.py , line in call cinder.volume.manager attempt attempt fn , attempt_number, false cinder.volume.manager file usr lib packages cinder volume targets tgt.py , line in create_iscsi_target cinder.volume.manager raise exception.iscsitargetcreatefailed volume_id=vol_id cinder.volume.manager iscsitargetcreatefailed: failed to create iscsi target for volume volume cinder.volume.manager",
        "where": 10
    },
    {
        "url": "https://stackoverflow.com/questions/68378228",
        "text": "parser argparse.argumentparser info, formatter_class=argparse.rawtexthelpformatter parser.add_argument t , token , openstack api token , args parser.parse_args user_token args.token headers content type : application json , authorization : x auth token user_token request url flavors : images : get account data try: result requests.get images , result_data json.loads result.content except exception as print sys.exit requesting please check conectivity images , if errors in result_data print sys.exit f token, please check token: result_data clean_agent agent.update agent_name openstack_images_%s agent_alias openstack_images description id clean_module modulo.update name openstack_image disk: , desc openstack image is result_data images mindisk , value str result_data images mindisk ,",
        "where": 9
    },
    {
        "url": "https://stackoverflow.com/questions/69113617",
        "text": "i: home systemctl status neutron openvswitch agent neutron openvswitch agent.service openstack neutron open vswitch plugin agent loaded: loaded lib systemd system neutron openvswitch agent.service; enabled; vendor preset: enabled active running since tue ist; ago process: sbin modprobe br_netfilter success process: bin chown neutron:neutron success process: bin mkdir p success main pid: neutron openvsw tasks: limit: cgroup: config etc neutron neutron.conf config etc neutron plugins openvswitch_age ovsdb client monitor tcp:127.0.0.1:6640 interface name,ofport,external_ids ovsdb client monitor tcp:127.0.0.1:6640 bridge name sep compute i ovs vsctl : as create manager ptcp:6640:127.0.0.1 compute i ovs vsctl : error: details : transaction causes multiple rows in table to have sep compute i ovs vsctl : error: details : transaction causes multiple rows in table to have iden sep compute i sudo : pam_unix sudo:session : session closed for user root sep compute i sudo : neutron var lib neutron usr bin neutron rootwrap ovs vsc sep compute i sudo : pam_unix sudo:session : session opened for user root by sep compute i ovs vsctl : as create manager ptcp:6640:127.0.0.1 sep compute i ovs vsctl : error: details : transaction causes multiple rows in table to have ident sep compute i ovs vsctl : error: details : transaction causes multiple rows in table to have iden sep compute i sudo : pam_unix sudo:session : session closed for user root",
        "where": 10
    },
    {
        "url": "https://stackoverflow.com/questions/69616030",
        "text": "import openstack conn openstack.connect region_name regionone auth_url domain_name project_name admin username admin password user_args name admin pasword aaa new password password_expires_at :none, links self enabled :true, domain_id original_password conn.identity.update_user user_args",
        "where": 3
    },
    {
        "url": "https://stackoverflow.com/questions/69672162",
        "text": "import openstack import openstack.compute.v2.limits import os import datetime getting limits conn openstack.connect auth_url=auth_url_domain, project_name= admin domain_name= root app cacrt cacrt ca.crt10.6.x.x.pem print data.to_dict",
        "where": 3
    },
    {
        "url": "https://stackoverflow.com/questions/71706595",
        "text": "systemctl status libvirtd libvirtd.service virtualization daemon loaded: loaded lib systemd system libvirtd.service; enabled; vendor preset: enabled active: active running since mon gmt; ago triggeredby: libvirtd admin.socket libvirtd.socket libvirtd ro.socket docs: man:libvirtd main pid: libvirtd tasks: limit: memory: cgroup: conf var lib libvirt dnsmasq leasefile ro conf var lib libvirt dnsmasq default.conf leasefile ro apr kypo virtual machine libvirtd : ethtool ioctl error: no such device apr kypo virtual machine libvirtd : ethtool ioctl error: no such device apr kypo virtual machine libvirtd : ethtool ioctl error: no such device apr kypo virtual machine libvirtd : ethtool ioctl error: no such device",
        "where": 10
    },
    {
        "url": "https://stackoverflow.com/questions/71937954",
        "text": "systemctl status cond no pager l cond devstack cond loaded: loaded etc systemd system cond enabled; vendor preset: enabled active: failed result: exit code since wed ago process: usr local bin nova conductor config file failure main pid: failure apr controller nova conductor : nova file usr local lib t packages identity base.py , line in get_endpoint_data apr controller nova conductor : nova service_catalog self.get_acces s session .service_catalog apr controller nova conductor : nova file usr local lib t packages identity base.py , line in get_access apr controller nova conductor : nova self.auth_ref self.get_auth_re f session apr controller nova conductor : nova file usr local lib t packages identity generic base.py , line in get_auth_ref apr controller nova conductor : nova self._plugin self._do_create_p lugin session apr controller nova conductor : nova file usr local lib t packages identity generic base.py , line in _do_create_plugin apr controller nova conductor : nova raise exceptions.discoveryfailur e apr controller nova conductor : nova iscoveryfailure: could not find versioned identity endpoints when attempting to authenticate. pl ease check that your auth_url is correct. unable to establish connection to identity: httpconnectionpool : max retries exceeded with url: identity caused by newconnectionerror object at failed to establish new connection: errno econnrefused apr controller nova conductor : nova",
        "where": 10
    },
    {
        "url": "https://stackoverflow.com/questions/72043799",
        "text": "openstack floating ip set usage: openstack floating ip set h port port fixed ip address ip address qos policy qos policy no qos policy tag tag no tag floating ip openstack floating ip set: error: too few arguments",
        "where": 10
    },
    {
        "url": "https://stackoverflow.com/questions/72181653",
        "text": "openstack image unset property hw_scsi_model badrequestexception: client for url: bad request openstack image show c status c visibility c properties field value properties hw_scsi_model= , os_hash_algo= status active visibility shared",
        "where": 10
    },
    {
        "url": "https://stackoverflow.com/questions/72372070",
        "text": "openstack help traceback most recent call last : file users library python openstack , line in module from openstackclient.shell import main file users library python python site packages openstackclient shell.py , line in module from openstack import utils file users library python python site packages openstack utils.py , line in module import queue importerror: no module named queue",
        "where": 10
    },
    {
        "url": "https://stackoverflow.com/questions/72435215",
        "text": "private osclient.osclientv3 authenticate try osclient.osclientv3 os osfactory.builderv3 .endpoint osproperties.getauthurl .credentials osproperties.getusername , osproperties.getpassword , identifier.byname osproperties.getdomainname .scopetoproject identifier.byid osproperties.getprojectid .authenticate ; os.useregion osproperties.getregion ; return os; catch responseexception exception log.debug with authentication os: , exception.getcause .getlocalizedmessage ; throw exception;",
        "where": 9
    },
    {
        "url": "https://stackoverflow.com/questions/72526356",
        "text": "etc su s c neutron db manage config file config file upgrade head neutron alembic.runtime.migration context impl mysqlimpl. alembic.runtime.migration will assume non transactional ddl. running upgrade for neutron ... alembic.runtime.migration context impl mysqlimpl. alembic.runtime.migration will assume non transactional ddl. alembic.runtime.migration running upgrade traceback most recent call last : file usr lib dist packages sqlalchemy engine base.py , line in _execute_context cursor, statement, parameters, context file usr lib dist packages sqlalchemy engine default.py , line in do_execute cursor.execute statement, parameters file usr lib dist packages pymysql cursors.py , line in execute result self._query query file usr lib dist packages pymysql cursors.py , line in _query conn.query q file usr lib dist packages pymysql connections.py , line in query self._affected_rows self._read_query_result file usr lib dist packages pymysql connections.py , line in _read_query_result result.read file usr lib dist packages pymysql connections.py , line in read first_packet self.connection._read_packet file usr lib dist packages pymysql connections.py , line in _read_packet packet.check_error file usr lib dist packages pymysql connections.py , line in check_error err.raise_mysql_exception self._data file usr lib dist packages pymysql err.py , line in raise_mysql_exception raise errorclass errno, errval pymysql.err.internalerror: cannot change column network_id : used in foreign key constraint subnets_ibfk_1 the above exception was the direct cause of the following exception: traceback most recent call last : file usr bin neutron db manage , line in module sys.exit main file usr lib dist packages neutron db migration cli.py , line in main return_val bool conf.command.func config, conf.command.name file usr lib dist packages neutron db migration cli.py , line in do_upgrade file usr lib dist packages neutron db migration cli.py , line in do_alembic_command getattr alembic_command, cmd config, file usr lib dist packages alembic command.py , line in upgrade script.run_env file usr lib dist packages alembic script base.py , line in run_env util.load_python_file self.dir, env.py file usr lib dist packages alembic util pyfiles.py , line in load_python_file module load_module_py module_id, path file usr lib dist packages alembic util compat.py , line in load_module_py spec.loader.exec_module module file frozen importlib._bootstrap_external , line in exec_module file frozen importlib._bootstrap , line in _call_with_frames_removed file usr lib dist packages neutron db migration alembic_migrations env.py , line in module run_migrations_online file usr lib dist packages neutron db migration alembic_migrations env.py , line in run_migrations_online context.run_migrations file string , line in run_migrations file usr lib dist packages alembic runtime environment.py , line in run_migrations self.get_context .run_migrations file usr lib dist packages alembic runtime migration.py , line in run_migrations step.migration_fn file usr lib dist packages neutron db migration alembic_migrations versions train expand line in upgrade existing_type=sa.string file string , line in alter_column file string , line in alter_column file usr lib dist packages alembic operations ops.py , line in alter_column return operations.invoke alt file usr lib dist packages alembic operations base.py , line in invoke return fn self, operation file usr lib dist packages alembic operations toimpl.py , line in alter_column file usr lib dist packages alembic ddl mysql.py , line in alter_column else existing_comment, file usr lib dist packages alembic ddl impl.py , line in _exec return conn.execute construct, file usr lib dist packages sqlalchemy engine base.py , line in execute return meth self, multiparams, params file usr lib dist packages sqlalchemy sql ddl.py , line in _execute_on_connection return connection._execute_ddl self, multiparams, params file usr lib dist packages sqlalchemy engine base.py , line in _execute_ddl compiled, file usr lib dist packages sqlalchemy engine base.py , line in _execute_context e, statement, parameters, cursor, context file usr lib dist packages sqlalchemy engine base.py , line in _handle_dbapi_exception util.raise_from_cause newraise, exc_info file usr lib dist packages sqlalchemy util compat.py , line in raise_from_cause reraise type exception , exception, file usr lib dist packages sqlalchemy util compat.py , line in reraise raise value.with_traceback tb file usr lib dist packages sqlalchemy engine base.py , line in _execute_context cursor, statement, parameters, context file usr lib dist packages sqlalchemy engine default.py , line in do_execute cursor.execute statement, parameters file usr lib dist packages pymysql cursors.py , line in execute result self._query query file usr lib dist packages pymysql cursors.py , line in _query conn.query q file usr lib dist packages pymysql connections.py , line in query self._affected_rows self._read_query_result file usr lib dist packages pymysql connections.py , line in _read_query_result result.read file usr lib dist packages pymysql connections.py , line in read first_packet self.connection._read_packet file usr lib dist packages pymysql connections.py , line in _read_packet packet.check_error file usr lib dist packages pymysql connections.py , line in check_error err.raise_mysql_exception self._data file usr lib dist packages pymysql err.py , line in raise_mysql_exception raise errorclass errno, errval oslo_db.exception.dberror: pymysql.err.internalerror cannot change column network_id : used in foreign key constraint subnets_ibfk_1 sql: alter table subnets modify network_id varchar not null background on this at: etc",
        "where": 10
    },
    {
        "url": "https://stackoverflow.com/questions/72721636",
        "text": "juju bootstrap bootstrap metadata simplestreams model model external model use floating microstack microstack debug juju.cmd supercommand.go:56 running juju gc debug juju.cmd supercommand.go:57 args: string snap juju bin juju , bootstrap , bootstrap , metadata simplestreams , model , , model , external , model , use floating , microstack , microstack , debug debug juju.cmd.juju.commands bootstrap.go:1307 authenticating with region and credential admin debug juju.cmd.juju.commands bootstrap.go:1455 provider attrs: map external network:external network:test policy target group: use secgroup:false use floating ip:true use openstack gbp:false cmd authkeys.go:114 adding contents of root .local share juju ssh juju_id_rsa.pub to authorized keys debug juju.cmd.juju.commands bootstrap.go:1530 preparing controller with config: map agent metadata url: agent stream:released apt ftp proxy: apt http proxy: apt https proxy: apt mirror: apt no proxy: authorized keys:ssh rsa juju client key automatically retry hooks:true backup dir: cloudinit userdata: container image metadata url: container image stream:released container inherit properties: container networking method: series:focal space: development:false disable network management:false disable telemetry:false egress subnets: enable os refresh update:true enable os upgrade:true external network:external fan config: firewall mode:instance ftp proxy: http proxy: https proxy: ignore machine addresses:false image metadata url: image stream:released juju ftp proxy: juju http proxy: juju https proxy: juju no proxy:127.0.0.1,localhost,::1 logforward enabled:false logging config: logging output: lxd snap channel:latest stable max action results age:336h max action results size:5g max status history age:336h max status history size:5g name:controller net bond reconfigure delay:17 network:test no proxy:127.0.0.1,localhost,::1 num container provision workers:4 num provision workers:16 policy target group: provisioner harvest mode:destroyed proxy ssh:false resource tags: snap http proxy: snap https proxy: snap store assertions: snap store proxy: snap store proxy url: ssl hostname verification:true test mode:false transmit vendor metrics:true type:openstack update status hook interval:5m use secgroup:false use floating ip:true use openstack gbp:false uuid:414283ed juju.provider.openstack provider.go:169 opening model controller warn juju.provider.openstack config.go:181 config attribute use floating ip is deprecated. you can instead use the constraint allocate public ip . debug juju.provider.openstack provider.go:979 authurl: debug juju.provider.openstack provider.go:979 authurl: debug goose logger.go:44 debug: auth details: token:gaaaaabis3o7i7xak0akxvqgm032h27etaab39ody8 tenantid:75a496aa41f944a4bd867557bcc7489d tenantname:admin userid:47980441e7bb47a69473b734091e19d5 domain: cmd bootstrap.go:855 creating juju controller microstack on microstack microstack debug goose logger.go:44 trace: api version will be inserted between and debug goose logger.go:44 debug: discovered api versions: version: major:2 minor:0 rel:self status:supported version: major:2 minor:1 rel:self status:current debug goose logger.go:44 trace: makeserviceurl: juju.cmd.juju.commands bootstrap.go:921 combined bootstrap constraints: debug juju.environs.bootstrap bootstrap.go:320 model controller supports application machine networks: true debug juju.environs.bootstrap bootstrap.go:322 network management by juju enabled: true debug juju.environs.bootstrap bootstrap.go:1065 no agent directory found, using agent metadata source: debug juju.environs.bootstrap bootstrap.go:1090 setting image metadata source: debug juju.environs.simplestreams simplestreams.go:423 searching for signed metadata in datasource bootstrap metadata debug juju.environs.simplestreams simplestreams.go:458 looking for data index using path streams debug juju.environs.simplestreams simplestreams.go:470 looking for data index using url file: root simplestreams images streams debug juju.environs.simplestreams simplestreams.go:473 streams not accessed, actual error: build snapcraft juju parts juju src environs simplestreams datasource.go:192: file: root simplestreams images streams not found debug juju.environs.simplestreams simplestreams.go:474 streams not accessed, trying legacy index path: streams index.sjson debug juju.environs.simplestreams simplestreams.go:489 cannot load index file: root simplestreams images streams index.sjson : file: root simplestreams images streams index.sjson not found debug juju.environs.simplestreams simplestreams.go:427 falling back to search for unsigned metadata in datasource bootstrap metadata debug juju.environs.simplestreams simplestreams.go:458 looking for data index using path streams debug juju.environs.simplestreams simplestreams.go:470 looking for data index using url file: root simplestreams images streams debug juju.environs.simplestreams simplestreams.go:473 streams not accessed, actual error: build snapcraft juju parts juju src environs simplestreams datasource.go:192: file: root simplestreams images streams not found debug juju.environs.simplestreams simplestreams.go:474 streams not accessed, trying legacy index path: streams index.json debug juju.environs.simplestreams simplestreams.go:493 read metadata index at file: root simplestreams images streams index.json debug juju.environs.simplestreams simplestreams.go:1025 finding products at path streams com.ubuntu.cloud released imagemetadata.json debug juju.environs imagemetadata.go:45 new user image datasource registered: bootstrap metadata juju.environs.bootstrap bootstrap.go:1127 custom image metadata added to search path cmd bootstrap.go:397 loading image metadata debug juju.environs imagemetadata.go:119 obtained image datasource bootstrap metadata debug juju.environs imagemetadata.go:119 obtained image datasource ubuntu cloud images debug juju.environs.bootstrap bootstrap.go:959 constraints for image metadata lookup microstack disco xenial opensuseleap eoan yakkety quantal trusty saucy raring genericlinux bionic vivid utopic kubernetes focal zesty wily precise impish cosmic artful jammy hirsute groovy armhf released debug juju.environs.simplestreams simplestreams.go:423 searching for signed metadata in datasource bootstrap metadata debug juju.environs.simplestreams simplestreams.go:458 looking for data index using path streams debug juju.environs.simplestreams simplestreams.go:470 looking for data index using url file: root simplestreams images streams debug juju.environs.simplestreams simplestreams.go:473 streams not accessed, actual error: build snapcraft juju parts juju src environs simplestreams datasource.go:192: file: root simplestreams images streams not found debug juju.environs.simplestreams simplestreams.go:474 streams not accessed, trying legacy index path: streams index.sjson debug juju.environs.simplestreams simplestreams.go:489 cannot load index file: root simplestreams images streams index.sjson : file: root simplestreams images streams index.sjson not found debug juju.environs.simplestreams simplestreams.go:427 falling back to search for unsigned metadata in datasource bootstrap metadata debug juju.environs.simplestreams simplestreams.go:458 looking for data index using path streams debug juju.environs.simplestreams simplestreams.go:470 looking for data index using url file: root simplestreams images streams debug juju.environs.simplestreams simplestreams.go:473 streams not accessed, actual error: build snapcraft juju parts juju src environs simplestreams datasource.go:192: file: root simplestreams images streams not found debug juju.environs.simplestreams simplestreams.go:474 streams not accessed, trying legacy index path: streams index.json debug juju.environs.simplestreams simplestreams.go:493 read metadata index at file: root simplestreams images streams index.json debug juju.environs.simplestreams simplestreams.go:497 skipping index file: root simplestreams images streams index.json because of missing information: index file has no data for cloud microstack not found debug juju.environs.bootstrap bootstrap.go:967 ignoring image metadata in bootstrap metadata: index file has no data for cloud microstack not found debug juju.environs.simplestreams simplestreams.go:423 searching for signed metadata in datasource ubuntu cloud images debug juju.environs.simplestreams simplestreams.go:458 looking for data index using path streams debug juju.environs.simplestreams simplestreams.go:470 looking for data index using url debug juju.environs.simplestreams simplestreams.go:473 streams not accessed, actual error: build snapcraft juju parts juju src environs simplestreams datasource.go:192: not found debug juju.environs.simplestreams simplestreams.go:474 streams not accessed, trying legacy index path: streams index.sjson debug juju.environs.simplestreams simplestreams.go:493 read metadata index at debug juju.environs.simplestreams simplestreams.go:497 skipping index because of missing information: index file has no data for cloud microstack not found debug juju.environs.bootstrap bootstrap.go:967 ignoring image metadata in ubuntu cloud images: index file has no data for cloud microstack not found debug juju.environs.bootstrap bootstrap.go:975 found image metadata from all image data sources debug goose logger.go:44 trace: makeserviceurl: cmd bootstrap.go:470 looking for packaged juju agent version for juju.environs.bootstrap tools.go:82 looking for bootstrap agent binaries: debug juju.environs.tools tools.go:87 finding agent binaries in stream: released debug juju.environs.tools tools.go:89 reading agent binaries with major.minor version debug juju.environs.tools tools.go:98 filtering agent binaries by version: debug juju.environs.tools tools.go:101 filtering agent binaries by os type: ubuntu debug juju.environs.tools tools.go:104 filtering agent binaries by architecture: debug juju.environs.tools urls.go:133 trying datasource keystone catalog debug juju.environs.simplestreams simplestreams.go:423 searching for signed metadata in datasource simplestreams debug juju.environs.simplestreams simplestreams.go:458 looking for data index using path streams debug juju.environs.simplestreams simplestreams.go:754 using candidate for content id com.ubuntu.juju:released:agents are mirrors:1.0 content download streams cpc mirrors agents.sjson debug juju.environs.simplestreams simplestreams.go:470 looking for data index using url debug juju.environs.simplestreams simplestreams.go:493 read metadata index at debug juju.environs.simplestreams simplestreams.go:1025 finding products at path streams com.ubuntu.juju released agents.sjson juju.environs.bootstrap tools.go:84 found packaged agent binaries cmd bootstrap.go:483 located juju agent version at warn juju.provider.openstack config.go:181 config attribute use floating ip is deprecated. you can instead use the constraint allocate public ip . cmd bootstrap.go:578 starting new instance for initial controller cmd bootstrap.go:167 launching controller instance s on microstack microstack... debug goose logger.go:44 trace: makeserviceurl: debug goose logger.go:44 trace: makeserviceurl: debug goose logger.go:44 trace: makeserviceurl: debug juju.environs.instances image.go:66 instance constraints region: microstack, series: bionic, arch: constraints: storage: juju.cmd.juju.commands bootstrap.go:884 failed to bootstrap model: cannot start bootstrap instance: no metadata for bionic images in microstack with arch debug juju.cmd.juju.commands bootstrap.go:885 details: build snapcraft juju parts juju src cmd juju commands bootstrap.go:984: failed to bootstrap model build snapcraft juju parts juju src environs bootstrap bootstrap.go:706: build snapcraft juju parts juju src environs bootstrap bootstrap.go:582: build snapcraft juju parts juju src provider common bootstrap.go:60: build snapcraft juju parts juju src provider common bootstrap.go:277: cannot start bootstrap instance build snapcraft juju parts juju src provider openstack provider.go:1078: build snapcraft juju parts juju src provider openstack provider.go:1099: build snapcraft juju parts juju src provider openstack image.go:87: build snapcraft juju parts juju src environs instances image.go:68: no metadata for bionic images in microstack with arch debug juju.cmd.juju.commands bootstrap.go:1641 cleaning up after failed bootstrap juju.provider.common destroy.go:21 destroying model controller juju.provider.common destroy.go:32 destroying instances debug goose logger.go:44 trace: makeserviceurl: debug goose logger.go:44 trace: makeserviceurl: debug juju.provider.openstack provider.go:1691 terminating instances debug goose logger.go:44 debug: auth details: token:gaaaaabis3o9i_bvbb1iv2oxxspzxjnbhdxvchi3kdql2w0e4roma4 tenantid:75a496aa41f944a4bd867557bcc7489d tenantname:admin userid:47980441e7bb47a69473b734091e19d5 domain: debug goose logger.go:44 trace: api version will be inserted between and debug goose logger.go:44 debug: discovered api versions: version: major:2 minor:0 rel:self status:current debug goose logger.go:44 trace: makeserviceurl: juju.provider.common destroy.go:56 destroying storage debug juju.provider.openstack cinder.go:119 volume url: debug goose logger.go:44 trace: makeserviceurl: debug goose logger.go:44 trace: makeserviceurl: debug goose logger.go:44 trace: makeserviceurl: debug goose logger.go:44 trace: makeserviceurl: cmd supercommand.go:544 command finished",
        "where": 10
    },
    {
        "url": "https://stackoverflow.com/questions/73075652",
        "text": "command: sudo cinder rootwrap env lc_all=c lvcreate t v n volume cinder volumes cinder volumes pool exit code: stdout: using stripesize kib.\\n stderr: device mapper: message ioctl on failed: operation not failed to process thin pool message delete failed to suspend cinder volumes cinder volumes pool with queued messages.\\n cinder.brick.local_dev.lvm traceback most recent call last : cinder.brick.local_dev.lvm file var lib kolla venv lib packages cinder brick local_dev lvm.py , line in create_volume cinder.brick.local_dev.lvm run_as_root=true cinder.brick.local_dev.lvm file var lib kolla venv lib packages os_brick executor.py , line in _execute cinder.brick.local_dev.lvm result self.__execute cinder.brick.local_dev.lvm file var lib kolla venv lib packages cinder utils.py , line in execute cinder.brick.local_dev.lvm return processutils.execute cinder.brick.local_dev.lvm file var lib kolla venv lib packages oslo_concurrency processutils.py , line in execute cinder.brick.local_dev.lvm cinder.brick.local_dev.lvm oslo_concurrency.processutils.processexecutionerror: unexpected while running command. cinder.brick.local_dev.lvm command: sudo cinder rootwrap env lc_all=c lvcreate t v n volume cinder volumes cinder volumes pool cinder.brick.local_dev.lvm exit code: cinder.brick.local_dev.lvm stdout: using stripesize kib.\\n cinder.brick.local_dev.lvm stderr: device mapper: message ioctl on failed: operation not failed to process thin pool message delete failed to suspend cinder volumes cinder volumes pool with queued messages.\\n cinder.brick.local_dev.lvm cinder.brick.local_dev.lvm cmd :sudo cinder rootwrap env lc_all=c lvcreate t v n volume cinder volumes cinder volumes pool: oslo_concurrency.processutils.processexecutionerror: unexpected while running command.",
        "where": 10
    },
    {
        "url": "https://stackoverflow.com/questions/73148077",
        "text": "import openstack from openstack import connection from requests import response from rest_framework import serializers from rest_framework import status openstack.enable_logging project_name= nomoresecret user_domain_id= project_domain_id= class serverserializer serializers.serializer instance_id serializers.charfield read_only=true name serializers.charfield max_length=100 image serializers.charfield max_length=500 fixed_ip serializers.ipaddressfield read_only=true floating_ip serializers.ipaddressfield allow_blank=true flavor serializers.charfield max_length=500 networks serializers.charfield max_length=500 key_pair serializers.charfield max_length=100, allow_blank=true status serializers.charfield max_length=20, read_only=true power_state serializers.charfield max_length=20, read_only=true console_url serializers.urlfield max_length=100, read_only=true def create self, validated_data servercreated conn.compute.create_server instance_name image_id=validated_data flavor_id=validated_data uuid validated_data network_id serverwait conn.compute.wait_for_server servercreated print serverwait.access_ipv4 return serverwait",
        "where": 3
    }
]